<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportCAD</class>
 <widget class="QWidget" name="ImportCAD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImportCAD</string>
  </property>
  <widget class="QLineEdit" name="lineEditFilePath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImportCAD">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditCADPreview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>400</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Columns</string>
   </property>
   <widget class="QListWidget" name="listWidgetSelected">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>141</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectColumn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveUpSelection">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveDownSelection">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Down</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetOption">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteSelection">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelPreView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSizeUnit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Size Unit:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSizeUnit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>um</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>inch</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
